BTMLColorLOSMod/SettingsColorHelper.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine;
+
+namespace BTMLColorLOSMod
+{
+    public static class SettingsColorHelper
+    {
+        // Make a new color from the floats r, g, b, a. 
+        // If the value is 0 <= X <= 1, we'll use that value 
+        // if the value is X > 1, we'll divide it by 255 to get the decimal value 
+        // and use that.
+        public static Color ColorFromValues(float r, float g, float b, float a)
+        {
+            float outR, outG, outB, outA;
+            if(r != 0 && r > 1)
+            {
+                outR = (float)(r / 255);
```

Comment:
Does this correctly box values? I would have guessed `(float)(5/255)` would end up being zero every time because of integer division rules. My first guess would have been `((float)r)/255`.

Reply:
Also seems like there could be a private `ColorFromValue` method that did the math being done 4 times here.

Reply:
It  correctly boxes values because the input is a float. If the input was an int, it'd be crappy. TBH I don't even know if float cast is necessary as the variable being set is a float.

Reply:
Agree on the second part. Makes it way cleaner.